apps/website/src/pages/docs/components.astro

```diff
@@ -1,16 +1,11 @@
 ---
+import { getCollection } from 'astro:content';
 import Layout from './_layout.astro';
 import Card from '~/components/Card.astro';
 
-const componentPages = (await Astro.glob('./*.mdx'))
-  .map(({ url, frontmatter }) => ({
-    url,
-    title: frontmatter.title,
-    description: frontmatter.description,
-    thumbnail: frontmatter.thumbnail,
-    group: frontmatter.group,
-  }))
-  .filter((page) => page.group);
+const componentPages = (
+  await getCollection('docs', ({ data: { group } }) => group !== 'intro')
+).map(({ slug, data }) => ({ url: `/docs/${slug}`, ...data }));
```

Comment:
I'm a little unfamiliar with what "slug" is. Looking at the types, it seems to be the component names.

![image](https://github.com/iTwin/iTwinUI/assets/45748283/059ace39-7072-40aa-b944-2ea989127918)

But looking at [this docs section](https://docs.astro.build/en/core-concepts/routing/#route-priority-order), I thought slug has something to do with a rest parameter for the path.

Reply:
slug is part of the url. in this case it simply maps to the file name. see https://docs.astro.build/en/guides/content-collections/#defining-custom-slugs

what you're linking to is the `[...]` syntax, which creates a catch-all dynamic route. so a single `[...slug].astro` file can be used to render all routes under `docs/`